src/main/webapp/app/shared/metis/posting-footer/post-footer/post-footer.component.ts

```diff
@@ -46,4 +69,32 @@ export class PostFooterComponent extends PostingFooterDirective<Post> implements
             this.tags = [];
         }
     }
+
+    /**
+     * sorts answerPosts by two criteria
+     * 1. criterion: resolvesPost -> true comes first
+     * 2. criterion: creationDate -> most recent comes at the end (chronologically from top to bottom)
+     */
+    sortAnswerPosts(): void {
+        if (!this.posting.answers) {
+            this.sortedAnswerPosts = [];
+            return;
+        }
+        this.sortedAnswerPosts = this.posting.answers.sort(
+            (answerPostA, answerPostB) =>
+                Number(answerPostB.resolvesPost) - Number(answerPostA.resolvesPost) || answerPostA.creationDate!.valueOf() - answerPostB.creationDate!.valueOf(),
+        );
+    }
+
+    /**
+     * creates empty default answer post that is needed on initialization of a newly opened modal to edit or create an answer post, with accordingly set resolvesPost flag
+     * @return AnswerPost created empty default answer post
+     */
+    createEmptyAnswerPost(): AnswerPost {
+        const answerPost = new AnswerPost();
+        answerPost.content = '';
+        answerPost.post = this.posting;
+        answerPost.resolvesPost = this.isAtLeastTutorInCourse;
```

Comment:
What exactly is the purpose of setting this here? Should this automatically resolve a post when a tutor answers it?
If this is the expected behavior then this doesn't work for me.
Also I don't think this would make all that much sense, as tutors might also ask follow up questions instead of directly answering the post.

If this is not the case feel free to tell me what the purpose should be.

Reply:
Okay. Maybe you can try to ask her or Stephan about this, as this might indicate broken and/or no longer needed functionality.
But this would not block this PR then.